src/main/java/seedu/address/model/friend/FriendId.java

```diff
@@ -11,9 +11,14 @@
 
     public static final String MESSAGE_EMPTY_FRIEND_ID =
             "FRIEND_ID cannot be empty.";
-    public static final String MESSAGE_INVALID_CHARACTERS = "FRIEND_ID provided must be a single word and only contain "
-            + "alphanumeric characters.";
-    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9._-]+$";
+    public static final String MESSAGE_INVALID_CHARACTERS = "FRIEND_ID provided must be a single alphanumeric word, "
+            + "of maximum 20 characters.";
```

Comment:
I am not sure why, but this sounds slightly weird to me. Perhaps we could rephrase it to be e.g.
"FRIEND_ID provided must be a single alphanumeric word, up to a maximum of 20 characters." ?